### PR TITLE
example: add user interaction sample(touch node instance).

### PR DESCRIPTION
**Pull request by taehyub**

### CODE REVIEW

**Comment by JSUYA** [comment]
Hi I made https://github.com/rive-app/rive-tizen/pull/56 for fix build test.
So please ignore the build fail of this PR

File: example/user_interaction_touch.cpp [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+       delete[] bytes;
+       fprintf(stderr, "failed to import %s\n", filename);
+       return;
+    }
+
+    artboard = file->artboard();
+    artboard->advance(0.0f);
+
+    delete animationInstance;
+    animationInstance = nullptr;
```
Comment (JSUYA) [inline example/user_interaction_touch.cpp:83]: This deletion is unnecessary

File: example/user_interaction_touch.cpp [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+
+    evas_object_image_pixels_dirty_set(view, EINA_TRUE);
+    evas_object_image_data_update_add(view, 0, 0, WIDTH, HEIGHT);
+
+    return ECORE_CALLBACK_RENEW;
+}
+
+static void runExample(uint32_t* buffer)
+{
+    std::string path = RIVE_FILE_DIR;
```
Comment (JSUYA) [inline example/user_interaction_touch.cpp:125]: this std:: namespace is unnecessary

**Review by JSUYA — COMMENTED** [review]
Please check my comment

**Review by hermet — APPROVED** [review]

**Review by hermet — CHANGES_REQUESTED** [review]
please check JSUYA comments.

File: example/user_interaction_touch.cpp (hunk 2 above)
Comment (taehyub) [reply example/user_interaction_touch.cpp:125]: I removed using namespace;

File: example/user_interaction_touch.cpp [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+    auto animation = artboard->animation(index);
+    if (animation) animationInstance = new rive::LinearAnimationInstance(animation);
+}
+
+static void loadRiveFile(const char* filename)
+{
+    lastTime = ecore_time_get();    //Check point
+
+    // Load Rive File
+    FILE* fp = fopen(filename, "r");
```
Comment (JSUYA) [inline example/user_interaction_touch.cpp:45]: this pointer don't closed.

File: example/user_interaction_touch.cpp [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+    fseek(fp, 0, SEEK_END);
+    size_t length = ftell(fp);
+    fseek(fp, 0, SEEK_SET);
+
+    uint8_t* bytes = new uint8_t[length];
+    if (fread(bytes, 1, length, fp) != length)
+    {
+       delete[] bytes;
+       fprintf(stderr, "failed to read all of %s\n", filename);
+       return;
```
Comment (JSUYA) [inline example/user_interaction_touch.cpp:56]: please add
fclose(fp);

File: example/user_interaction_touch.cpp [hunk 5]
```diff
@@ -0,0 +1,210 @@
...
+    }
+
+    auto reader = rive::BinaryReader(bytes, length);
+    rive::File* file = nullptr;
+    auto result = rive::File::import(reader, &file);
+    if (result != rive::ImportResult::success)
+    {
+       delete[] bytes;
+       fprintf(stderr, "failed to import %s\n", filename);
+       return;
```
Comment (JSUYA) [inline example/user_interaction_touch.cpp:66]: please add
fclose(fp);

File: example/user_interaction_touch.cpp [hunk 6]
```diff
@@ -0,0 +1,210 @@
...
+    }
+
+    artboard = file->artboard();
+    artboard->advance(0.0f);
+
+    auto animation = artboard->firstAnimation();
+    if (animation) animationInstance = new rive::LinearAnimationInstance(animation);
+
+    delete[] bytes;
+}
```
Comment (JSUYA) [inline example/user_interaction_touch.cpp:76]: please add
fclose(fp);

**Review by JSUYA — CHANGES_REQUESTED** [review]
I write additional comments about file pointer. Please check again

**Comment by JSUYA** [comment]
If you have edited the code for the reviewer's comment, or if the problem has been resolved, click `Resolve conversation` to close the comment.

**Review by JSUYA — APPROVED** [review]

**Pull request merged**